test/e2e/network/service.go

```diff
@@ -3283,42 +3282,72 @@ var _ = SIGDescribe("ESIPP [Slow]", func() {
 			framework.Failf("Service HealthCheck NodePort still present")
 		}
 
-		endpointNodeMap, err := jig.GetEndpointNodes()
+		config := e2enetwork.NewNetworkingTestConfig(f, false, false)
```

Comment:
Please don't create it here - move it down to where it really starts being used.